tests/bin/ds-container-remove.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
+
+docker rm -f ds
```

Comment:
This script it unused. Was it meant to replace e.g. lines 146-148 in the old diff, the "Remove DS" jobs?